guardrail.ini

```diff
@@ -37,10 +39,10 @@ dead_code_ignore =
 # Comma seperated language if more than one
 programming_language = python
 [options]
-# option can be either trur or false
+# option can be either true or false
 linting=true
-cpd=true
-coverage=true
+cpd=false
```

Comment:
Why are the below values changed to false please?

Reply:
this file is not anymore used